tests/pytest/run_tests.py

```diff
@@ -13,6 +13,7 @@ def get_parser():
     parser.add_argument("-c", "--config-version", dest="config_version", default="3.8", help="Set config version: [3.8]")
     parser.add_argument("-i", "--install-dir", dest="install_dir", help="Custom install path for syslog-ng, under this directory should be exist: [bin/, etc/, sbin/, var/]")
     parser.add_argument("-l", "--log-level", dest="log_level", default="INFO", help="Set minimum debug log level: [INFO, ERROR]")
+    parser.add_argument("-d", "--test-subdir", dest="test_subdir", default="functional_tests", help="Query tests from subdir")
```

Comment:
don't forget to port `func_test/modules/python`, because it does not run currently this way